esx_service/vmdk_ops.py

```diff
@@ -87,6 +87,7 @@
 import error_code
 import auth_api
 from error_code import ErrorCode
+import re
```

Comment:
never mind .. not sure why diff is saying it is added as part of this change. Please ignore above comment. Sorry for the confusion.

Thanks!